.browserslistrc

```diff
@@ -0,0 +1,3 @@
+Defaults
+ie >= 11
+last 2 versions
```

Comment:
Will the "last 2 versions" always be the same, assuming we run it with the same version of babel listed in dependencies?  In general, we want to make our builds reproducible, so if we build the same code now or a year from now, we'll get exactly the same result.

Reply:
Negative, the last 2 versions will build relative to the current browser versions. The web standard for this kind of thing is called [browserlist](https://github.com/browserslist/browserslist). If you like we can discuss which browsers to target, but I don't think we should try to make our builds reproducible. Otherwise we can never take into account new browsers / versions / features (Or by supporting old browsers that no one uses, the code will grow really big, really fast)

Reply:
It is worth noting, that since IE is no longer going, that this will always include IE10 & IE11 (Last 2 versions will never change)